dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -272,17 +272,23 @@ EditWordBreakProcW(LPWSTR lpch, INT index, INT count, INT code)
         {
             if (index)
                 --index;
-            while (index && !IsWordBreak(lpch[index]))
+            while (index &&
```

Comment:
Remove this

Reply:
@cbialorucki As the code now currently stands in the PR, why would Maj need to remove the `index` (!= 0) test?
Because, if `index` was originally == 1, it gets decreased to zero just above (line 274), and then in this while-loop, we would look at stuff like `lpch[-1]` etc.

Reply:
@HBelusca I think he meant to say that I should revert this to `while (index && !IsWordBreak(lpch[index]))` and use his suggestion to return index - 1 if it's not 0 or 0 if index is 0 using return index ? index - 1 : index;

Reply:
ok i see :+1: